services/director-v2/tests/unit/test_modules_dynamic_sidecar_docker_service_specs.py

```diff
@@ -75,22 +79,316 @@ def simcore_service_labels() -> SimcoreServiceLabels:
     )
 
 
+@pytest.fixture
+def minimal_catalog_config(project_env_devel_environment, monkeypatch):
+    """set a minimal configuration for testing the director connection only"""
+
+
+@pytest.fixture
+def expected_dynamic_sidecar_spec() -> dict[str, Any]:
```

Comment:
maybe we could have this one in a separate mock file?

Reply:
well to be honest I had to use that to be sure I would not break the code.